testing_test.go

```diff
@@ -70,7 +70,7 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func startPostgres() (container testcontainers.Container, dn *sql, err error) {
+func startPostgres() (container testcontainers.Container, db *sql, err error) {
```

Comment:
Nice catch but unrelated, please make a separate PR for this if you want, otherwise i will do